lib/enrich.ts

```diff
@@ -0,0 +1,100 @@
+import { EMBED_JSONLD_CONTEXT } from '../config';
+import { OPARL_JSON_LD_CONTEXT } from '../constants';
+import { removeVersionFromOparlSchemaUri, rewriteLinkWithProxy } from './utils';
+
+export function enrichOparlDataToJsonLd(oparlData: any, proxyUrl: string) {
```

Comment:
unexpected any, typescript complains if you don't remove this (but i agree that we shouldn't specify the full oparl type)